tests/run_tox.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python
+"""
+Proxy script from tox. This is an intermediate script so that we can setup
+the environment properly then call ceph-ansible for testing, and finally tear
+down, while keeping tox features of simplicity and combinatorial confgiruation.
+
+NOTE: Do not run this script directly as it depends on a few environment
+variables that tox will set, like ceph-ansible's scenario path
+"""
+import logging
+import os
+import pwd
+import shutil
+import subprocess
+import time
+
+
+logging.basicConfig()
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+
+
+def sh(cmd, cwd=None, ignore_errors=False, capture=False):
```

Comment:
I suppose we could do that if we're ok with `check_call` merging `stdout` and `stderr`